xrayvision/visibility.py

```diff
@@ -48,7 +50,7 @@ def time_range(self) -> Optional[Iterable[Time]]:
 
     @property
     @abc.abstractmethod
-    def vis_labels(self) -> Optional[Iterable[str]]:
+    def vis_labels(self) -> Sequence[Iterable[str]]:
```

Comment:
Since `vis_labels` isn't required, should this be `Optional[Sequence[str]]`?  Or perhaps `Optional[Sequence[Iterable[str]]]`?